pilot/pkg/model/context.go

```diff
@@ -927,7 +931,7 @@ func (node *Proxy) SetGatewaysForProxy(ps *PushContext) {
 }
 
 func (node *Proxy) SetServiceInstances(serviceDiscovery ServiceDiscovery) {
```

Comment:
nit: Should make the naming consistent `s/instances/targets`?

Reply:
done